Tasks/my_queue.py

```diff
@@ -19,7 +21,12 @@ def dequeue():
 
 	:return: dequeued element
 	"""
-	return None
+	if len(my_qu) == 0:
+		return print("My_stack is empty")
+	returned = my_qu[0]
+	my_qu.pop(0)
```

Comment:
Вообще функция pop(0) возвращает тот элемент, который она выкинула из списка (в отличие от del my_qu[0]), поэтому можно было написать вот так:
```suggestion
	returned = my_qu.pop(0)
	return returned
```